include/mbgl/map/map_options.hpp

```diff
@@ -88,6 +88,24 @@ class MapOptions final {
      */
     bool crossSourceCollisions() const;
 
+    /**
+     * @brief Specify whether render data for layers, sources and images should be kept between renderStill() calls.
+     * If set to false, all the data is created from scratch for every renderStill() call.
+     * By default, it is set to true.
```

Comment:
By default, it is set to true unless on TileMode

Reply:
Scrap that, got confused.

Reply:
The idea is that this flag is set explicitly, when needed. So that we obviate regressions e.g. in snapshotter cc @alexshalamov